libqtile/layout/base.py

```diff
@@ -29,10 +29,6 @@
 
 class Layout(CommandObject, configurable.Configurable, metaclass=ABCMeta):
     """This class defines the API that should be exposed by all layouts"""
-    @classmethod
-    def _name(cls):
-        return cls.__class__.__name__.lower()
-
     defaults = [(
         "name",
```

Comment:
I guess we should drop this as well?

Reply:
I think this gets it in the defaults generated by the docs (albeit with a value of "None", so maybe not all that useful), I could go either way.

Reply:
I think it's just noise in the docs

Reply:
Always happy to delete more code :)